src/router/route.tsx

```
@@ -0,0 +1,26 @@
+import { createBrowserRouter } from 'react-router';
+
+import { AppLayout } from '@layout';
+import { Error, Home, NotFound } from '@pages';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)